### ISSUE TITLE
Premove on Mobile is Not Equal to Premove on Website

### ISSUE THREAD

**Issue by S1M0N38** [lichess-org/mobile#740]
I noticed a small difference between lichess.org and the mobile app (v0.8.17) when premoving pieces. I play using drag & drop on both mobile and browser.

## On lichess.org

On the website, I can make a premove. While the premove is active, I can pick up another piece to be ready to move it. Effectively, I've set up a premove and then prepared:

- For the move after the premove (if the premove is played)
- For an alternative move (if the premove is not played, e.g., a premoved capture cannot be played)

## On Mobile

I tried to replicate the same on the mobile app. I set up a premove. When I grab a new piece to be ready for when the premove is resolved, the premove gets canceled.

---

To recap, on mobile, I cannot grab a new piece until the premove is resolved; otherwise, it gets canceled. This does not occur on the browser.

I think this difference might be negligible in most situations but could become advantageous for players using the browser in time scramble situations.

**Comment by veloce** [comment]
Right, it is important to have the same behaviour here.

**Comment by sldenazis** [comment]
+1 on this one, the old app has this function and it's crucial for faster time controls.
